driver/controller.go

```diff
@@ -839,7 +837,14 @@ func (d *Driver) ControllerExpandVolume(ctx context.Context, req *csi.Controller
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "ControllerExpandVolume could not retrieve existing volume: %v", err)
 	}
-	ll.Info("extracting storage from requested capacity range")
+
+	ll := d.log.WithFields(logrus.Fields{
```

Comment:
Can we call this `log` instead of `ll`? 😅

Reply:
Good idea -- massively renamed this everywhere. :)